docs/setting-up/client/postgresql.md

```diff
@@ -190,6 +190,13 @@ You can now [add the service](#add-service).
 
     You can get a list of other available settings with `SELECT * FROM pg_stat_monitor_settings;`.
 
+    Another useful settings parameter is:
+    ```ini
+    pg_stat_monitor.pgsm_normalized_query
+    ```
+
+     If the value is set to 1, then query actual values are replaced by placeholders. If the value is 0, you can see the examples in QAN. 
```

Comment:
```suggestion
    If the value is set to 1, the actual query values are replaced by placeholders. If the value is 0, the examples are given in QAN. 
```

Reply:
Can we give a link to QAN where the examples are there?

Reply:
Examples are in detail of specific row, so direct link is always different depends on opened row. If you want we can describe it somehow. Its when you open row details then there is tab called examples.

Reply:
@rasika-chivate Take a look on message above. Is it OK?

Reply:
I think we should describe it.

Reply:
Examples can be found in QAN details tab example.

Is this OK? Or better ideas how to describe it?

Reply:
Added, resolved.